kernel/aster-nix/src/net/socket/ip/options.rs

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use crate::impl_socket_options;
+
+impl_socket_options!(
+    pub struct RetOpts(bool);
+    pub struct RecvErr(bool);
+    pub struct RecvTTL(bool);
```

Comment:
Unless there is a purpose for consistency with existing code, prefer `RecvTtl` to `RecvTTL`.

This avoids the awkward naming of `XMLHTTPRequest` (which should be `XmlHttpRequest` instead).

For example, the Rust standard library uses [TcpStream](https://doc.rust-lang.org/std/net/struct.TcpStream.html), not `TCPStream`.